jest.config.setup.files.after.env.js

```diff
@@ -33,8 +33,11 @@ const allowedErrors = [
   /^Error: Expected key descriptor but found "" in ""/, // This is a false negative in @testing-library/user-event
 ];
 
-console.error = (message) => { // eslint-disable-line
-  if (!allowedErrors.some(error => error.test(message))) {
-    throw new Error(message);
-  }
+const originalError = window.console.error;
```

Comment:
I realized we weren't formatting the hijacked error message. Using a real console.error here will allow an error message to actually render as it is intended.